CIP-0072/README.md

```diff
@@ -0,0 +1,413 @@
+                      "type": "string"
+                    },
+                    "auditType": {
+                      "type": "string"
+                    },
+                    "signature": {
+                      "type": "string"
+                    },
+                    "publicKey": {
+                      "type": "string"
+                    }
+                  },
+                  "required": [
+                    "auditId",
+                    "auditor",
+                    "auditLink",
+                    "auditType",
+                    "signature",
+                    "publicKey"
+                  ]
+                }
+              ]
+            }
+          },
+          "required": [
+            "id",
+            "name",
+            "purpose",
+            "type",
+            "versions",
+            "audits"
+          ]
+        }
+      ]
+    }
+  },
+  "required": [
+    "subject",
+    "projectName",
+    "link",
+    "twitter",
+    "category",
+    "subCategory",
+    "description",
+    "releases",
+    "scripts"
+  ]
+}
+```
+
+## Example
+
+```json
+{
+    "subject": "9SYAJPNN",
+    "projectName": "My Project",
+    "link": "https://myProject.app",
+    "twitter": "https://twitter.com/MyProject",
+    "category": "GAMING",
+    "subCategory": "RPG",
+    "description": {
+                   "short": "A story rich game where choices matter"
+    },	
+
+    "releases": [
+        {
+            "releaseNumber": 1,
+            "releaseName": "V1",
+            "auditId": "z5L90f",
+            "scripts": [
+                {
+                    "id": "PmNd6w",
+                    "version": 1
+                }
+            ]
+        }
+    ],
+    "scripts": [
+        {
+            "id": "PmNd6w",
+	    "name": "Marketplace",
+            "purpose": "SPEND",
+	    "type": "PLUTUS",		
+            "versions": [
+                {
+                    "version": 1,
+		    "plutusVersion": 1, 
+	            "fullScriptHash": "711dcb4e80d7cd0ed384da5375661cb1e074e7feebd73eea236cd68192",                     
+                    "scriptHash": "1dcb4e80d7cd0ed384da5375661cb1e074e7feebd73eea236cd68192",
+                    "contractAddress": "addr1wywukn5q6lxsa5uymffh2esuk8s8fel7a0tna63rdntgrysv0f3ms"
+                }    
+            ],
+          "audits": [
+              {
+                "auditId": "z5L90f",
+                "auditor": "Canonical LLC.",
+                "auditLink": "https://github.com/somlinkToAessment",
+                "auditType": "MANUAL",
+                "signature": "0x1234567890abcdef",
+                "publicKey": "0x1234567890abcdef"
+              }
+          ]
+        }
+    ]
+}
+```
+
+### **Scripts Audits**
+As part of the metadata we will be including a list of audits for each script. This will be a list of audits that have been performed on the script. The audits will be performed by a third party auditor and will be signed by the auditor. 
```

Comment:
```suggestion
As part of the metadata we will be including a list of audits for each DApp. This will be a list of audits that have been performed on the deployed scripts comprising the DApp. The audits will be performed by a third party auditor and will be signed by the auditor. 
```